PCSX2_suite.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30225.117
```

Comment:
I'd exclude these changes, or drop it to 15 since we support VS 2017 too.

Reply:
It isn't the minimum version. 
https://docs.microsoft.com/en-us/visualstudio/extensibility/internals/solution-dot-sln-file?view=vs-2019

Read docs.

Reply:
Yeah this change is inconsequential. The solution will still open for previous versions.